src/main/java/com/google/devtools/build/lib/rules/java/ResourceJarActionBuilder.java

```diff
@@ -115,6 +116,11 @@ public void build(JavaSemantics semantics, RuleContext ruleContext, String execG
       command.addExecPaths("--classpath_resources", classpathResources);
     }
 
+    ImmutableMap.Builder<String, String> executionInfo = ImmutableMap.builder();
```

Comment:
Could you please change this to create a new map iff `ruleContext.isAllowTagsPropagation()`is true?